src/helm/config/model_deployments.yaml

```diff
@@ -1041,6 +1041,14 @@ model_deployments:
   ## GPT 3.5 Turbo Models
   # ChatGPT: https://openai.com/blog/chatgpt
 
+  - name: openai/gpt-3.5-turbo-instruct
+    model_name: openai/gpt-3.5-turbo-instruct
+    tokenizer_name: openai/cl100k_base
+    max_sequence_length: 4096
+    max_sequence_length: 4097
```

Comment:
Are we still doing this off by one? I thought we would get rid of it

Reply:
Doing this for consistency for now, since `gpt-3.5-turbo-instruct` uses the same completions API as `text-davinci-003` and we did that for `text-davinci-003`. I should check with @teetone whether we can just delete this.